.github/workflows/server_tests.yaml

```diff
@@ -0,0 +1,61 @@
+name: server-tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  server-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+#        python-version: [3.7, 3.8, 3.9, 3.10]
```

Comment:
@dongreenberg you think it's overkill to run these tests across multiple python versions?

Reply:
Will they launch in parallel (i.e. as separate actions)?